sdk/purview/Azure.Analytics.Purview.Scanning/src/Generated/PurviewClassificationRuleClient.cs

```diff
@@ -71,6 +71,20 @@ public PurviewClassificationRuleClient(Uri endpoint, string classificationRuleNa
 
         /// <summary> Get a classification rule. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <example>
+        /// This sample shows how to call GetPropertiesAsync and parse the result.
+        /// <code><![CDATA[
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("<https://my-account-name.azure.com>");
+        /// var client = new PurviewClassificationRuleClient(endpoint, credential);
```

Comment:
[ClassificationRuleClient is a subclient](https://apiview.dev/Assemblies/Review/705ac828d1604e8881431011883e2f58#Azure.Analytics.Purview.Scanning.PurviewScanningServiceClientOptions) and should be created using

`new PurviewScanningServiceClient(endpoint, credential).GetClassificationRuleClient("<classification-rule-name>");`

Reply:
fixed by using `CodeGenClient` attribute, the side effect is that `PurviewClassificiationRuleClient` to `PurviewClassificiationRule`.

Reply:
Whats happens?

Reply:
@Pablo1q just some name changes on the generated classes. Since the SDK is not GA, it's OK for breaking changes.